tests/test_crypto/test_fetchai.py

```diff
@@ -639,3 +644,62 @@ def test_execute_shell_command(mock_api_call):
 
     res = cosmos_api._execute_shell_command(["test", "command"])
     assert res == {"SOME": "RESULT"}
+
+
+def send_remaining_funds_back_to_faucet(account: FetchAICrypto):
+    """Sends remaining funds back to faucet"""
+    faucet_address = "fetch193vvag846gz3pt3q0mdjuxn0s5jrt39fsjrays"
+    fetchai_api = FetchAIApi(**FETCHAI_TESTNET_CONFIG)
+    balance = fetchai_api.get_balance(account.address)
+    assert balance is not None, "balance is of type None"
+
+    tx_fee = 1000
```

Comment:
Is this the minimum required tx fee? Please define a constant above and use here (e.g. `tx_fee = MINIMUM_TX_FEE`).